css/media.css

```diff
@@ -300,15 +300,12 @@
   .project-details-popup {
     overflow: auto;
     box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
-    display: none;
-  }
-
-  .popup {
     height: 100vh;
     width: 80.3vw;
-    position: fixed;
-    top: 7rem;
+    top: 3rem;
     right: 7.3rem;
+    position: fixed;
```

Comment:
- [ ] Kindly rework this style because the issue highlighted by the previous reviewer still remains. 😄 🙏

Reply:
Hi, @MahmoudBakr23 was the position not fixed from your end? Because it is from my end. Sorry, can you explain it a little bit more?

Reply:
Hi @kifle23 
The previous reviewer asked to fix the problem of scrolling as shown in the video above, this issue still remains, please fix it 🙏

Reply:
@MahmoudBakr23, okay thank you, brother